UMAProject/Assets/UMA/Editor/TreeElementUtility.cs

```diff
@@ -0,0 +1,318 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework;
```

Comment:
Style points for testing. 

But putting tests directly into the code is dangerous. I don't recall what unity version where they disabled the NUnit.Framework by default and on some unity versions it's not playing nice with Assembly Definitions.

So even though this is in an editor folder I'd prefer a separate file and indeed a folder dedicated to this.

Reply:
Removed tests to avoid possible issues with NUnit.